dry-view.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.1.0'
 
+  spec.add_runtime_dependency "inflecto"
```

Comment:
I just used inflecto by habit/reflex. I'll switch this across to dry-core's inflector :)